csrc/scheduler/pointwise.cpp

```diff
@@ -369,8 +369,10 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
   // not used. should allow to use both unroll and vectorization together in
   // heuristics tuning.
   if (params->vectorization_factor == 1) {
-    params->unroll_factor = scheduler_utils::safeDiv(
+    auto total_unroll = scheduler_utils::safeDiv(
         max_vect_unroll_factor, params->vectorization_factor);
+    params->unroll_factor_inner = total_unroll;
```

Comment:
IIUC, this PR shouldn't impose any functional changes. So I would expect all old use of `params->unroll_factor` to be replaced with `params->unroll_factor_inner`.

Reply:
Yes. here all the unroll factors go to `unroll_factor_inner` through `params->unroll_factor_inner = total_unroll;`